utils/createToken.js

```diff
@@ -0,0 +1,12 @@
+const jwt = require('jsonwebtoken');
+const { super_secret } = require('../utils/secrets');
+
+const createToken = ({ user_id, name, hotel_id, chat_id }) =>
```

Comment:
Might want to remove chat_id here as it'l be undefined for staff.

Reply:
Done 👍 , thanks!